tdms/tests/unit/hdf5_and_tdms_objects/test_hdf5_Cuboid.cpp

```diff
@@ -22,6 +23,8 @@ TEST_CASE("HDF5: Read Cuboid") {
   Cuboid cube;
 
   SECTION("Read into existing object") {
+    if (!std::filesystem::exists(tdms_object_data))
+      SKIP(cant_find_test_data_message);
```

Comment:
This snippet is appearing quite a lot and likely will continue to appear quite a lot going forward -> function in the unit test utils namespace?  Would then save us another variable declaration too as `cant_find_test_data_message` only needs to appear in this function too.

Does Catch even allow this and behave properly if we do it?

```c++
void skip_test_if_data_missing(const std::string &expected_data_file) {
if (!std::filesystem::exists(expected_data_file)) {
    SKIP(cant_find_test_data_message);
}
}
```

Reply:
🍉